include/openmc/cell.h

```diff
@@ -11,7 +11,7 @@
 #include "hdf5.h"
 #include "pugixml.hpp"
 #ifdef DAGMC
-#include "DagMC.hpp"
+#include "dagmc.h"
```

Comment:
is this a bespoke part of openmc?

Reply:
The `DagMC.hpp` include is, but `dagmc.h` isn't. This `#ifdef` can be removed then I think. 
Thanks!